.github/workflows/lint-prettier.yml

```diff
@@ -32,7 +32,8 @@ jobs:
           pnpm install --frozen-lockfile
       - name: Analyze TypeScript files
         run: |
-          pnpm recursive run lint
```

Comment:
Can you look at #838 - Antoine figured out how to bring down the time it takes eslint in the ci/cd pipeline down to about 6-7min - 20+minutes is insanely frustrating.  He ended up adding the .eslintrc to each project, for some reason having only 1 was making the eslint go hunting for references for a long time. Adding them to each project dropped each from 20-30 sec to 5-6 seconds each.  Is this a good approach?  How do we ensure each project has the same eslint settings?  DO we add a script to auto-addupdate these as projects are added?